worker/src/main/scala/org/apache/celeborn/service/deploy/worker/WorkerSource.scala

```diff
@@ -84,6 +84,7 @@ class WorkerSource(conf: CelebornConf) extends AbstractSource(conf, Role.WORKER)
   addTimer(CLEAN_EXPIRED_SHUFFLE_KEYS_TIME)
 
   addHistogram(FETCH_CHUNK_TRANSFER_SIZE)
+  addHistogram(PARTITION_FILE_SIZE)
```

Comment:
It's a histogram, you can add the mean, max, 99th, and  95th values into the grafana dashboard.